packages/hardhat-zksync-solc/src/constants.ts

```diff
@@ -0,0 +1,28 @@
+import { ZkSolcConfig } from './types';
+
+export const ZK_ARTIFACT_FORMAT_VERSION = 'hh-zksolc-artifact-1';
+export const LATEST_VERSION = '1.2.1';
+
+export const defaultZkSolcConfig: ZkSolcConfig = {
+    version: LATEST_VERSION,
+    compilerSource: 'binary',
+    settings: {
+        compilerPath: '',
+        experimental: {},
+    },
+};
+
+export const SUPPORTED_ZKSOLC_CONTRACT_OUTPUT_SELECTIONS = [
+    'abi',
+    'evm.methodIdentifiers',
+    'storageLayout',
+    'irOptimized',
```

Comment:
This one is really dangerous.
Passed to solc 0.7 or older, it will almost certainly crash the compiler.
This is probably part of the reason why AAVE crashes.
Anyway, let's not restrict it. It must be up to the user and will not be a problem if there are no other bugs elsewhere.